docs/contributor/resources/03-moduletemplate.md

```diff
@@ -106,6 +106,12 @@ By default, it will most likely be easiest to use [Kyma CLI](https://github.com/
 The `mandatory` field indicates whether the module is installed in all runtime clusters without any interaction from the user.
 Mandatory modules do not appear in the Kyma CR `.status` and `.spec.modules`, furthermore they have the same configuration across all runtime clusters.
 
+
+### **.spec.associatedResources**
+
+The `associatedResources` field is a list of module related resource definitions that usually should be cleaned up when uninstalling a module. 
```

Comment:
```suggestion
The `associatedResources` field is a list of module-related custom resource definitions (CRDs) that should be cleaned up during module deletion. 
```
should be cleaned up but are not? like leftovers? or just should be in any case? Can we say deletion? In the end-user docs we always say "add" or "delete" a module. Or maybe in that context "uninstalling" is better?

Reply:
It's an information for SRE in terms of deletion being stuck to have an indicator for what resources to look for on clean-up. KLM is not doing anything with it.